packages/mdc-icon-button/foundation.js

```diff
@@ -34,48 +34,27 @@ class MDCIconButtonToggleFoundation extends MDCFoundation {
 
   static get defaultAdapter() {
     return {
-      addClass: (/* className: string */) => {},
-      removeClass: (/* className: string */) => {},
-      setText: (/* text: string */) => {},
-      getAttr: (/* name: string */) => /* string */ '',
-      setAttr: (/* name: string, value: string */) => {},
-      notifyChange: (/* evtData: IconButtonToggleEvent */) => {},
+      addClass: () => {},
+      removeClass: () => {},
+      hasClass: () => {},
+      setAttr: () => {},
+      notifyChange: () => {},
     };
   }
 
   constructor(adapter) {
     super(Object.assign(MDCIconButtonToggleFoundation.defaultAdapter, adapter));
 
-    const {ARIA_PRESSED} = MDCIconButtonToggleFoundation.strings;
+    const {ICON_BUTTON_ON_CLASS} = MDCIconButtonToggleFoundation.cssClasses;
 
     /** @private {boolean} */
-    this.on_ = this.adapter_.getAttr(ARIA_PRESSED) === 'true';
-
+    this.on_ = this.adapter_.hasClass(ICON_BUTTON_ON_CLASS) || false;
```

Comment:
Why `|| false`? Shouldn't `hasClass` return a boolean anyway?

Also, is there a reason we even need the `on_` variable? Could we just implement `isOn` to check whether this class is present?